service.js

```diff
@@ -101,8 +111,8 @@ module.exports = function setup(serviceConfig) {
           logger.error(`${serviceConfig.getBaseUrl()} [do_not_track] could not parse response: ${response.text}`);
           return callback(`${serviceConfig.getBaseUrl()} [do_not_track] could not parse response: ${response.text}`);
         } else {
-          logger.error(`${synthesizeUrl(serviceConfig, req)} could not parse response: ${response.text}`);
-          return callback(`${synthesizeUrl(serviceConfig, req)} could not parse response: ${response.text}`);
+          logger.error(`${synthesizeUrl(serviceConfig, req, res)} could not parse response: ${response.text}`);
```

Comment:
would be nicer to save the results of `synthesizeUrl` instead of calling it twice in consecutive lines of code